doc/download_model.md

```diff
@@ -0,0 +1,33 @@
+# Download Your Trained Model
+
+Once you have trained your model with SQLFlow you may need to download your
```

Comment:
Once we train a model using SQLFlow via a statement like the following,

```sql
SELECT ... TO TRAIN # Please put a realistic statement here.
```

the trained model goes to the model zoo.  Anyone with the read access can write SQLFlow statement to visually explain the model or to use the model for prediction.

In some cases, you might want to export and download a trained model from the model zoo, so you can use it out of SQLFlow, for example, load it to an online prediction service of an online advertising system.  To export a model, you can use the command-line tool `sqlflow`.
we can download the trained model, if necessary, using the